apps/api/src/modules/project/schema/project.schema.ts

```diff
@@ -1,14 +1,16 @@
 import {Type} from '@sinclair/typebox';
 import {Nullable} from '../../../common/typebox/nullable';
 
+export const BaseResponse = {
+  id: Type.String({format: 'uuid'}),
+  name: Type.String(),
+  createdAt: Type.Unsafe<Date | string>({format: 'date-time'}),
+  updatedAt: Type.Unsafe<Date | string>({format: 'date-time'}),
+  ownerId: Type.String({format: 'uuid'}),
+};
```

Comment:
i think we are missing the `version` property that the client will use to parse the given data. Also we can remove the `ownerId`